src/TodoList.jsx

```diff
@@ -1,19 +1,25 @@
 import TodoListItem from "./TodoListItem"
 
-function TodoList({ todoList }) {
+function TodoList({ todoList, onCompleteTodo }) {
+  const filteredTodoList = todoList.filter((todo) => !todo.isCompleted);
+  todoList = filteredTodoList;
```

Comment:
This works but there is a clear distinction in `todoList` and `filteredTodoList` which is good to preserve. By overwriting `todoList` you add extra code that is unnecessary. Best to pass `filteredTodoList` to the JSX. This clearly conveys what that data in this component is and how it changes.